backend/tournesol/views/video.py

```diff
@@ -154,13 +159,24 @@ def get_queryset(self):
             for crit in settings.CRITERIAS
         ]
         criteria_weight = Case(*criteria_cases, default=0)
-        queryset = (
-            queryset.annotate(
-                total_score=Sum(F("criteria_scores__score") * criteria_weight)
-            )
-            .filter(total_score__gt=0)
-            .order_by("-total_score")
+
+        unsafe = request.query_params.get('unsafe')
+        show_unsafe = False
+        if unsafe:
+            show_unsafe = unsafe == 'true'
```

Comment:
(minor) Can be simplified for readibility:
```suggestion
        show_unsafe = request.query_params.get('unsafe') == 'true'
```